packages/shared/src/common/validate-uuid.ts

```diff
@@ -0,0 +1,5 @@
+const uuidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
+
+export function isValidUuid(uuid: string) {
```

Comment:
does the underlying ,library we use to generate the uuid not have a function that does this? just double checking

Reply:
I don't see it there..